snap/info_test.go

```diff
@@ -1174,6 +1174,12 @@ func (s *infoSuite) TestExpandSnapVariables(c *C) {
 	c.Assert(info.ExpandSnapVariables("$SNAP_DATA/stuff"), Equals, "/var/snap/foo/42/stuff")
 	c.Assert(info.ExpandSnapVariables("$SNAP_COMMON/stuff"), Equals, "/var/snap/foo/common/stuff")
 	c.Assert(info.ExpandSnapVariables("$GARBAGE/rocks"), Equals, "/rocks")
+
```

Comment:
Can you please add a comment that explains why this, perhaps unexpected, result is indeed correct for how we use instances? Something like this:
```
// NOTE: Despite setting the instance key the variables expand to the same
// value as before. This is because they are used from inside the mount namespace
// of the instantiated snap where the mount backend will ensure that
// the regular (non-instance) paths contain instance-specific code and data.
```

Reply:
s/NOTE://... Every comment is a note.